propagation/trace_context.go

```diff
@@ -28,6 +28,7 @@ const (
 	maxVersion        = 254
 	traceparentHeader = "traceparent"
 	tracestateHeader  = "tracestate"
+	parentDelimiter   = "-"
```

Comment:
```suggestion
	delimiter   = "-"
```

This is a delimiter of the trace-context, it doesn't have anything to do with the trace parent, right?